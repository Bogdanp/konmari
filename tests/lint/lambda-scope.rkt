#lang racket/base

(lambda _
  (void))

(lambda _
  (void))

(lambda (a)
  a)

(lambda (a b)
  a)
