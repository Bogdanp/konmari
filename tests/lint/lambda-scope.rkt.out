/lambda-scope.rkt:12:12:warning:identifier 'b' is never used
